heimdall-core/src/main/java/br/com/conductor/heimdall/core/util/ConstantsPath.java

```diff
@@ -37,6 +37,8 @@ private ConstantsPath() { }
      
      public static final String PATH_API = "/v1/api";
 
+     public static final String APIID = "/{apiId}";
```

Comment:
Change constant name to `API_ID_URL_PATH_VAR`

Reply:
Done